environment.yml

```diff
@@ -0,0 +1,15 @@
+name: unidist
+channels:
+  - conda-forge
+dependencies:
+  - cloudpickle
+  - dask[complete]>=2.22.0
+  - distributed>=2.22.0
+  - mpi4py>=3.0.3
+  - msgpack-python
+  - packaging
+  - pandas
+  - pickle5
```

Comment:
Is it possible to check python version for this package?

Reply:
Not sure that is possible. Users that have python>=3.8 are not able to install deps from the files, they get the following error:
```
UnsatisfiableError: The following specifications were found
to be incompatible with the existing python installation in your environment:

Specifications:

  - pickle5 -> python[version='>=3.6,<3.7.0a0|>=3.7,<3.8.0a0']

Your python: python=3.8.5

```
We should probably exclude pickle5 from the files and add a user-friendly error message if the package is not installed. What do you think?

Reply:
You can specify python version for package using this:
1) for requirements.txt - https://www.python.org/dev/peps/pep-0508/
2) for environment.yml - https://docs.conda.io/projects/conda-build/en/latest/resources/package-spec.html

What OS do you use to get this error? I can't get this error on Ubuntu:
```
pickle5                   0.0.11                   pypi_0    pypi
pillow                    8.4.0                    pypi_0    pypi
pip                       21.2.4           py38h06a4308_0
pluggy                    1.0.0                    pypi_0    pypi
prometheus-client         0.12.0                   pypi_0    pypi
protobuf                  3.19.1                   pypi_0    pypi
psutil                    5.8.0                    pypi_0    pypi
py                        1.11.0                   pypi_0    pypi
py-spy                    0.3.11                   pypi_0    pypi
pyasn1                    0.4.8                    pypi_0    pypi
pyasn1-modules            0.2.8                    pypi_0    pypi
pyparsing                 2.4.7                    pypi_0    pypi
pyrsistent                0.18.0                   pypi_0    pypi
pytest                    6.2.5                    pypi_0    pypi
python                    3.8.12               h12debd9_0
python-dateutil           2.8.2                    pypi_0    pypi
pytz                      2021.3                   pypi_0    pypi
```

Reply:
> You can specify python version for package using this:
> 
> for requirements.txt - https://www.python.org/dev/peps/pep-0508/
> for environment.yml - https://docs.conda.io/projects/conda-build/en/latest/resources/package-spec.html

Yes, we can but that won't work. See below.

> What OS do you use to get this error? I can't get this error on Ubuntu:

You do not get the error since you install pickle5 from PyPI on Ubuntu. pip is able to build .so for pickle5 for python 3.8 on Ubuntu but I am not sure that it works with python>3.8.3. I get the error both on Ubuntu and on Windows when installing pickle5 from conda-forge. That happens because conda-forge doesn't have pickle5 build for python 3.8. Other than that, I get a more strange error when installing pickle5 from PyPI on Windows. So I suggest going the way I mentioned above.

Reply:
In that case I agree with the way mentioned above.

Reply:
Updated the changes